plugins/catalog-backend/src/modules/core/BuiltinKindsEntityProcessor.ts

```diff
@@ -298,6 +298,12 @@ export class BuiltinKindsEntityProcessor implements CatalogProcessor {
         RELATION_OWNED_BY,
         RELATION_OWNER_OF,
       );
+      doEmit(
+        domain.spec.subdomainOf,
```

Comment:
Do you have particular preferences about `subdomainOf` (which has preexisting examples like components' `subcomponentOf`), vs `parent` (which groups uses)? No need to change anything, just asking for now.

Reply:
I don't have a particular preference, I'm happy with either. @janogonzalez ?